### ISSUE TITLE
WIP: [Sourcing][Accessibility][Multi input Mobile mode dialog]Popup contains unnecessary regions

### ISSUE THREAD

**Issue by I543348** [SAP/fundamental-ngx#10299]
#### Is this a bug, enhancement, or feature request?
Bug

#### Briefly describe your proposal.
The popup contains unnecessary regions (e.g. <footer> at the end): It was not error, only a note and good to have. Regions should only be used to structure large content areas. Regions are output by the screen reader. In a small pop-up, it is rather annoying if the button is focused at the end and a region is also output.

#### If this is a bug, please provide steps for reproducing it.
Regions should only be used to structure large content areas. Regions are output by the screen reader. In a small pop-up, it is rather annoying if the button is focused at the end and a region is also output.
tested on multi input value help dialog footer area

#### Is there anything else we should know?
reported by customer

**Comment by droshev** [comment]
@I543348 Can you provide more context information and screenshots?

**Comment by I543348** [comment]
Hi @droshev ,This is a customer testing issue.
We had discussed this and triaged the issue along with ATL.
Could you please connect with them for more details?

Thanks.

**Comment by droshev** [comment]
@I543348 The agreement is that you provide us with all the input so we can work on the issue.

**Comment by nikolay-kolarov** [comment]
Hi,

I checked the dialog (MultiInput on Mobile -> value help) and there is a footer element, which is a landmark element and forms a region for Jaws https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA11.
In this case the header even is missing otherwise the customer might have complained that there are two regions.

Still, this is an on-purpose decision to have a header and footer in the dialogs in order to have a common consistent structure. And this is used across UI5 and UI5 WebComponents dialogs.

So I think we can close this one.

Best Regards,
Nikolay

**Comment by I543348** [comment]
ATL Re-verification needed

**Comment by sagar7259** [comment]
Hello, 

This can be documented as a note and close this issue has this is an on-purpose decision to have a header and footer in the dialogs in order to have a common consistent structure. And this is used across UI5 and UI5 WebComponents dialogs has per the comment from Nikolay.

Regards,
Sagar
